packages/component-meta/scripts/generateComponentMeta.ts

```diff
@@ -107,7 +123,7 @@ const generateComponentMeta = (): void => {
   }, {} as Record<TagName, string>);
 
   const meta: ComponentsMeta = TAG_NAMES.reduce((result, tagName) => {
-    const source = componentSourceCode[tagName];
+    let source = componentSourceCode[tagName];
```

Comment:
I think we shouldn't modify the original. Instead keep it as it as and have the modified one in addition and only use it where needed.